Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderWorker.cpp

```diff
@@ -27,10 +27,11 @@
 #include <ScriptCanvas/Grammar/AbstractCodeModel.h>
 #include <ScriptCanvas/Results/ErrorText.h>
 #include <ScriptCanvas/Utils/BehaviorContextUtils.h>
-#include <ScriptCanvas/Core/Core.h>
 
 namespace ScriptCanvasBuilder
 {
+    using namespace ScriptCanvas;
+
```

Comment:
Repeat: the `using namespace ScriptCanvas;` call needs to be moved to function scope or removed altogether